packages/backend/server/package.json

```diff
@@ -147,7 +147,7 @@
     }
   },
   "nodemonConfig": {
-    "exec": "node",
+    "exec": "node --loader ts-node/esm --experimental-specifier-resolution=node",
```

Comment:
`node --import ./scripts/register.js`

Reply:
I tried and found this would lead to regression of the ` Unknown file extension ".ts"` error. Updating docs to use `affine server dev` seems to be better.